Content.Server/Fluids/Components/MopComponent.cs

```diff
@@ -99,11 +105,6 @@ async Task<bool> IAfterInteract.AfterInteract(AfterInteractEventArgs eventArgs)
                 return false;
             }
 
-            if (CurrentVolume <= 0)
-            {
-                Owner.PopupMessage(eventArgs.User, Loc.GetString("mop-component-mop-is-dry-message"));
-                return false;
-            }
```

Comment:
I removed this error message that plays when a dry mop clicks a puddle. In the new behaviour, the dry mop should try to pick up solution from the puddle.